sigstore/_cli.py

```diff
@@ -261,6 +273,12 @@ def _parser() -> argparse.ArgumentParser:
         default=os.getenv("SIGSTORE_CERT_OIDC_ISSUER"),
         help="The OIDC issuer URL to check for in the certificate's OIDC issuer extension",
     )
+    verification_options.add_argument(
+        "--offline",
+        action="store_true",
+        default=_boolify_env("SIGSTORE_OFFLINE"),
```

Comment:
When you integrate with TUF, will this flag be respected? TUF by default will make network calls to fetch the latest timestamp if expired. We might want to scope this to only Rekor.

Reply:
Or make explicit a default exception for TUF: TUF really can't be done totally offline (or guaranteed offline, unless you want to accept some verification failure here that you might have expired TUF metadata if you can't have a network call.)